scripts/finalize-template.py

```diff
@@ -11,43 +11,40 @@ def get_parser():
         "Fills out argument templates, then destroys, saves the new files,"
         "and destroys all template files, including this script."
     )
-    parser.add_argument('--app-name', required=True)
-    parser.add_argument('--maintainer', required=True)
-    parser.add_argument('--keepalive', required=False, default=False, action='store_true', dest='meta_keepalive')
+    parser.add_argument("--app-name", required=True)
+    parser.add_argument("--maintainer", required=True)
+    parser.add_argument("--keepalive", required=False, default=False, action="store_true", dest="meta_keepalive")
     return parser
 
 
 def get_template_files():
-    return [
-        str(path.joinpath()) for path in list(Path('.').rglob('*.template.*'))
-    ]
+    return [str(path.joinpath()) for path in list(Path(".").rglob("*.template*"))]
```

Comment:
Changed the trailing ".*" to "*" since for globbing purposes the former REQUIRED a "." which basically meant it would only work on files with extensions ... which `Dockerfile` lacks